### ISSUE TITLE
Cannot connect via eth or wifi   v1.11.1

### ISSUE THREAD

**Issue by DG12** [ruuvi/ruuvi.gateway_esp.c#496]
Firmware version : v1.11.1

eth plugged in

My network is 192.168.1.*

with 
Reset button is pressed during boot - clear settings in flash
or 
The CONFIGURE button has been released - restart system

full log at http://MyBeacons.info/screenlog.220315.0728.txt
Firmware version : v1.11.1l)

I (2521) ETH: [main] Ethernet init
I (2525) ETH: [main] ethernet_update_ip
I (2529) ETH: [main] Using ETH DHCP
I (2534) ETH: [main] ETH ip updated
I (2538) ETH: [main] esp_eth_driver_install
I (2563) esp_eth.netif.glue: 94:b9:7e:59:a6:63
I (2563) esp_eth.netif.glue: ethernet attached to netif
...
I (2657) ruuvi_gateway: [main] Firmware auto-updating is active, run next check after 2400 seconds
W (11481) ADV_POST_TASK: [adv_post_task] Can't send advs, no network connection
W (11481) ADV_POST_TASK: [adv_post_task] Can't send statistics, no network connection
...
I (12657) ruuvi_gateway: [main] free heap: 131132
I (62589) ruuvi_gateway: [main] MAIN_TASK_SIG_DEACTIVATE_WIFI_AP
I (62589) wifi_manager: [main] wifi_manager_stop_ap
I (62590) wifi_manager: [wifi_manager] MESSAGE: ORDER_STOP_AP
I (62597) wifi_manager: [wifi_manager] Configure WiFi mode: Station
I (62604) wifi:mode : sta (94:b9:7e:59:a6:60)
I (62608) wifi_manager: [sys_evt] WIFI_EVENT_AP_STOP
I (62614) ETH: [main] Ethernet start
I (64218) ETH: [sys_evt] Ethernet Started
I (64221) ETH: [sys_evt] Ethernet Link Up
I (64221) ETH: [sys_evt] Ethernet HW Addr 94:B9:7E:59:A6:63
I (64223) ruuvi_gateway: [sys_evt] Ethernet connection established
I (64230) ETH: [main] Set hostname for Ethernet interface: RuuviGatewayFCC3
I (64238) LEDS: [main] leds_indication_network_no_connection
I (64244) LEDS: [main] LED: Start blinking, interval: 200 ms, duty cycle: 50%
I (64253) ruuvi_gateway: [main] free heap: 132376
W (71481) ADV_POST_TASK: [adv_post_task] Can't send advs, no network connection
W (71481) ADV_POST_TASK: [adv_post_task] Can't send statistics, no network connection

default IP for WiFi AP: 10.10.0.1

+++++++
I (2476) ruuvi_gateway: [main] WiFi country after wifi_init: CC=CN , max_tx_power=11 dBm, schan=1, nchan=13, policy=AUTO
I (2486) wifi:set country: **cc=FI^@ sc**han=1 nchan=13 policy=0
+++country code may have JUNK

PS displaying first and/or last character of passwords would be helpful without causing a security risk.

**Comment by DG12** [comment]
Successful log from 22-01-15 at http://MyBeacons.info/ FCC3.220115

**Comment by ojousima** [comment]
Is your unit one of the beta hardware versions? We had some units with faulty Ethernet connector, those would establish link to the router but never receive IP address. Your log reminds me of those units

**Comment by DG12** [comment]
Now wait a minute!! After power cycle, power cycle with button pressed, button presses for 10 seconds many times,
I disconnected USB from laptop and connected it to power block, still nothing.

A few hours later, I go to check my "fing" (it's a network monitor) to see if, when and for how long  the FCC3 access point was up and suddenly the Access Point for the gateway(RuuviGatewayFCC3) is there.

I switch my MacBook Pro to the access point and I get the GUI page-welcome screen (configuration wizard) via "startup browser connection" i.e. without entering address, at 10.10.0.1

I accidentally close safari :-(.

 When I restart safari (no change to network) I get a not quite FINNished TEXT version of page-welcome!
See http://MyBeacons.info/gateWayFCC3_page-welcome-txt.png ???

I think I refreshed and got the GUI version back (very strange)

I get the page-software_update page since I have V1.11.1 and latest is V1.11.2 And I go for it! :-) It completes normally.
Complete wifi details SSID, PW .
Choose wifi accessPoint and get 192.168.1.30. :-)
12:26:09 All page-finished. , 
but wait:
12:27:15 Oh no! connection to the Ruuvi Gateway is lost.  :-(
12:27:47 MacBook shows acess point but could not be joined :-(
++++++++++++
Now 13:39 Access point  RuuviGatewayFCC3 is not available! %-{
RuuviGatewayFCC3 is at 192.168.1.30  , connecting to it I get the not quite FINNished TEXT page, refresh and GUI page-welcome appears!

It seems to be collecting packets:
13:48:20  ruuvigw_received_advertisements 53982	
13:49:50 ruuvigw_received_advertisements 55537
13:53:08 ruuvigw_received_advertisements 56639


I have screen shots for TEXT version of page-welcome, 12:26:09,  12:27:15 and 12:27:47  if you want them
